engine/src/audio/CodecRegistry.cpp

```diff
@@ -57,22 +57,23 @@ namespace Audio {
             codecPriority[codec] = priority;
             nameCodec[codec->getName()] = codec;
 
-            BOOST_LOG_TRIVIAL(trace) << "Registering codec " << codec->getName().c_str();
+            // static initializer ordering issue between boost logger and other code?
+            std::cerr << "Registering codec " << codec->getName().c_str();
```

Comment:
FYI - we're moving away from using this methodology to using the BOOST_LOG_TRIVIAL everywhere we can; the exception being the code before BOOST_LOG can get setup.

Reply:
Yeah - I thought I was having an initializer ordering issue with the logger (because where it was failing, before main, didn't make much sense). However, reverting this back works just fine on the release target, so this was just another instance of the debug target being broken (with internet browsing for similar errors seeming to indicate the likely culprit being the boost libraries being compiled with a different iterator debugging level flag than the rest of the code. This causes failures in things like boost internal stringcopy functions and other inscrutable sadness). I'm making a pass over this and the other comments and should have an update committed later today.